api/responses/members_in_cd.php

```diff
@@ -0,0 +1,76 @@
+<?php
+/*
+    Copyright (C) 2004-2010 Kestas J. Kuliukas / Timothy Jones
+
+	This file is part of webDiplomacy.
+
+    webDiplomacy is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Affero General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    webDiplomacy is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU Affero General Public License
+    along with webDiplomacy.  If not, see <http://www.gnu.org/licenses/>.
+ */
+namespace webdiplomacy_api;
+
+defined('IN_CODE') or die('This script can not be run by itself.');
+
+require_once ('game_country.php');
+
+/**
+ * Games in CD JSON response
+ * @package webdiplomacy_api
+ */
+class CountriesInCivilDisorder {
+	/**
+	 * List of (gameID, countryID)
+	 * @var array
+	 */
+	public $value = array();
+
+	/**
+	 * Load the countries in CD;
+	 */
+	function load()
+	{
+		global $DB;
+
+		$countryTabl = $DB->sql_tabl("SELECT m.gameID, m.countryID
+                                      FROM wD_Members AS m
+                                      LEFT JOIN wD_Orders AS o ON ( o.gameID = m.gameID AND o.countryID = m.countryID )
+                                      LEFT JOIN wD_Games AS g ON ( g.id = m.gameID )
+                                      WHERE NOT o.id IS NULL
+                                            AND g.variantID in (1, 15, 23)
+                                            AND g.phaseMinutes >= 10
+                                            AND m.missedPhases > 0
+                                            AND g.processStatus = 'Not-processing'
+                                            AND g.phase IN ('Diplomacy', 'Retreats', 'Builds')
+                                            AND g.processTime >= UNIX_TIMESTAMP()

```

Comment:
UNIX_TIMESTAMP() looks unfamiliar, can we use time() or whatever is used elsewhere in the code? Otherwise there just might be some weird time zone business if the database defaults to something different. (Unlikely, but a quick fix)